### PR TITLE
Removed duplicated utilities in data-transport-layer

### LINKED ISSUE #2109: Remove duplicated utilities in data-transport-layer
The `data-transport-layer` appears to be duplicating some utilities that already exist in `core-utils`. All of the utilities [within this file](https://github.com/ethereum-optimism/optimism/blob/develop/packages/data-transport-layer/src/utils/common.ts) can be deleted and instead imported from `@eth-optimism/core-utils`. Should be a really straightforward issue and will slightly improve test coverage by deleting unnecessary repeated code.

### PR DESCRIPTION
**Pull request by tonykogias**
**Description**
Removed duplicated utilities in data-transport-layer

**Metadata**
- Fixes #2109

### CODE REVIEW

**Review by smartcontracts — COMMENTED** [review]
Great work, thank you! Just need to fix the linting issue and [add a changeset](https://github.com/ethereum-optimism/optimism/blob/develop/CONTRIBUTING.md#changesets) for the DTL.

**Comment by tonykogias** [comment]
> Great work, thank you! Just need to fix the linting issue and [add a changeset](https://github.com/ethereum-optimism/optimism/blob/develop/CONTRIBUTING.md#changesets) for the DTL.

Done! Let me know if there are any more issues

File: packages/data-transport-layer/src/utils/common.ts [hunk 1]
```diff
@@ -1,16 +1,3 @@
...
- *
- * @param ms Number of milliseconds to sleep.
- */
-export const sleep = async (ms: number): Promise<void> => {
-  return new Promise<void>((resolve) => {
-    setTimeout(resolve, ms)
-  })
-}
-
 export const assert = (condition: () => boolean, reason?: string) => {
```
Comment (smartcontracts) [inline packages/data-transport-layer/src/utils/common.ts:1]: I think you can also delete this function, which means you should be able to delete the entire file

Comment (tonykogias) [reply packages/data-transport-layer/src/utils/common.ts]: You are right, I didn't notice `assert ` function is not used in that package.

**Comment by tynes** [comment]
This needs a merge conflict, could you squash your commits into a single commit while you are at it? Thank you!

**Comment by tonykogias** [comment]
I will open up a new one because I messed this up.

**Pull request closed without merging**